.github/workflows/trigger-homebrew-update.yml

```
@@ -0,0 +1,21 @@
+  trigger-tap-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Trigger tap repository update"
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ secrets.TAP_REPO_PAT }}
+          repository: mostlygeek/homebrew-llama-swap
+          event-type: new-release
+          client-payload: |-
+            {
+              "release": { 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)